src/schema/common.cljs

```diff
@@ -8,7 +8,20 @@
 (def ^:private ?customization-color
   [:or :string :keyword])
 
+(def ^:private ?public-key
+  [:re #"^0x04[0-9a-f]{128}$"])
+
+(def ^:private ?general-rpc-call
+  [:cat
+   [:map
```

Comment:
I understand your usage of the RPC effect in this PR is not using all possible keys to the map, but since you're defining the RPC schema, I think it would be beneficial to define the other keys too (`:js-response`, etc). I documented them all in the docstring of `json-rpc.events/call`.